### ISSUE TITLE
Support for MC1.9

### ISSUE THREAD

**Issue by kriation** [Zyin055/zyinhud#55]
Huge fan of this mod, and would love to see support for 1.9.

**Comment by kriation** [comment]
I'm set up my development environment, and have started addressing the differences. I'll send a pull request in a new branch if/when I can get a successful build against 1.9.

**Comment by liushuyu** [comment]
Hi, I have done a quick fix against ZyinHUD ( For Minecraft 1.9). There are still many bugs and limitations. I will upload my change sets ASAP for your reference. Thx

EDIT: Uploaded: https://github.com/liushuyu/zyinhud/tree/1.9-test

Known issues & limitations:
- [X] Sound system broken (Low Health Monitor won't work) **[Fixed]**
- [ ] Poor support for Main & Off hands (New in MC 1.9)  
- [X] Torch Aid won't work **[Fixed]**
- [ ] Weapon Aid works in a strange way
- [ ] And more...

**Comment by tegsirat** [comment]
Amazing mod - can someone build against 1.9.4? I do not know how to do this.

**Comment by cyilin** [comment]
@tegsirat I have updated to 1.9.4
https://github.com/cyilin/zyinhud/releases
Thanks @liushuyu

**Comment by liushuyu** [comment]
> @tegsirat I have updated to 1.9.4
> https://github.com/cyilin/zyinhud/releases

Great job! :+1:

**Comment by tegsirat** [comment]
Amazing!

Thank you so much!

**Comment by HeroCC** [comment]
Is there a 1.9 build? Either official or a fork.

**Comment by tegsirat** [comment]
Yes, there’s one for 1.9 & 1.9.4. Both are forks, and do have some issues, but are amazing to have!

1.9 = https://www.dropbox.com/s/rcsbqh64wclsow3/ZyinsHUD-1.9-v1.5.0-SNAPSHOT.jar?dl=0

1.9.4 = https://www.dropbox.com/s/fwmphh1biu31vqv/ZyinsHUD-.1.9.4.-v.1.5.0-SNAPSHOT.jar?dl=0

Thanks to the folks that compiled these!

Is there a 1.9 build? Either official or a fork.

—

**Comment by tegsirat** [comment]
anybody mind building against 1.10?

**Comment by HoldYourWaffle** [comment]
@cyilin has a 1.10-snapshot build

**Comment by HeroCC** [comment]
@cyilin since Zyin hasn't been around for a while, mind enabling the ticket tracker on your fork? I'd report a bug (#61) but it isn't enabled, and would like you to take a peek if possible.

**Comment by cyilin** [comment]
@HeroCC OK, I will try to fix this bug
